tests/device_tests/test_autolock.py

```diff
@@ -70,7 +71,7 @@ def test_apply_minimal_auto_lock_delay(client):
     Verify that the delay is not below the minimal auto-lock delay (10 secs)
     otherwise the device may auto-lock before any user interaction.
     """
-    set_autolock_delay(client, 1 * 1000)
+    set_autolock_delay(client, 10 * 1000)
```

Comment:
this testcase is now pointless, it was checking that the autolock delay is at least 10 seconds.

please remove it completely.

Reply:
Done 8516e7a.